views/js/test/previewer/runner/test.js

```diff
@@ -22,14 +22,34 @@
 define([
     'jquery',
     'lodash',
-    'core/promise',
     'taoQtiTestPreviewer/previewer/runner',
     'json!taoQtiItem/test/samples/json/space-shuttle.json',
     'lib/jquery.mockjax/jquery.mockjax',
     'css!taoQtiTestPreviewer/previewer/provider/item/css/item'
-], function($, _, Promise, previewerFactory, itemData) {
+], function($, _, previewerFactory, itemData) {
     'use strict';
 
+    const providers = {
+        runner : {
+            id: 'qtiItemPreviewer',
+            module: 'taoQtiTestPreviewer/previewer/provider/item/item',
+            bundle: 'taoQtiTestPreviewer/loader/qtiPreviewer.min',
+            category: 'previewer'
+        },
+        proxy: {
+            id: 'qti',
+            module: 'taoQtiTest/runner/proxy/qtiServiceProxy',
+            bundle: 'taoQtiTest/loader/qtiTestRunner.min',
```

Comment:
This should be instead:
```
id: 'qtiItemPreviewerProxy',
module: 'taoQtiTestPreviewer/previewer/proxy/item',
bundle: 'taoQtiTestPreviewer/loader/qtiPreviewer.min',
```

Reply:
Does it matter ? I can replace it but I thought that "any" provider can be set without breaking the test

Reply:
I'm afraid it matters indeed. When I run the tests, it blocks on every test that reference the wrong proxy.

Reply:
I know why, I was still loading the correct proxy on https://github.com/oat-sa/extension-tao-testqti-previewer/pull/25/commits/ccde0e2e140ed2e8f9c3e2ab5fb65776654d6790#diff-a974ff9e0fa2cd0d2806828835f6f877L60
Now it's fixed